mlir/include/mlir/Tools/mlir-opt/MlirOptMain.h

```diff
@@ -229,6 +229,13 @@ class MlirOptMainConfig {
 /// the loaded IR.
 using PassPipelineFn = llvm::function_ref<LogicalResult(PassManager &pm)>;
 
+/// Register all dialects and parse command line options.
```

Comment:
What does "Register all dialects" refer to here?

Reply:
> What does "Register all dialects" refer to here?

Oops, wrong comment, should be `Register and parse command line options` here.

Reply:
(not fixed yet)

Reply:
Yep, should be fixed now 🙏